commit 4c98afbd97f190e740ac7127f9af99b5333a741a
Author: Alex Fiestas <afiestas@kde.org>
Date:   Thu Feb 2 16:07:38 2012 +0100

    Adding eDP port as "laptop port" just like LVDS
    
    eDP are just like LVDS but more modern, so they should be
    considered as a laptop screen and inhibit when only it is
    available.
    
    This will fix the inhibition problem probably for the rest
    of the affected users.
    
    amichair thanks for the debugging :)
    
    CCBUG:289760

diff --git a/kcontrol/randr/module/randrmonitor.cpp b/kcontrol/randr/module/randrmonitor.cpp
index f1a42f3..e83d6b3 100644
--- a/kcontrol/randr/module/randrmonitor.cpp
+++ b/kcontrol/randr/module/randrmonitor.cpp
@@ -213,7 +213,7 @@ void RandrMonitorModule::checkInhibition()
     Q_FOREACH(const QString monitor, activeMonitorsList) {
         //LVDS is the default type reported by most drivers, default is needed because the
         //NVIDIA binary blob always report default as active monitor.
-        if (!monitor.contains("LVDS") && !monitor.contains("default")) {
+        if (!monitor.contains("LVDS") && !monitor.contains("default") && !monitor.contains("eDP")) {
             inhibit = true;
             break;
         }
